apps/learning-snippets/src/test/tree-widget/FilteredPaths.test.tsx

```diff
@@ -24,18 +24,28 @@ import { getSchemaContext, getTestViewer, mockGetBoundingClientRect, TestUtils }
 
 import type { SelectionStorage } from "@itwin/unified-selection";
 import type { IModelConnection, Viewport } from "@itwin/core-frontend";
+import { useCallback } from "react";
 
 // __PUBLISH_EXTRACT_START__ TreeWidget.GetFilteredPathsComponentExample
 type useModelsTreeProps = Parameters<typeof useModelsTree>[0];
+type GetFilteredPathsType = Exclude<useModelsTreeProps["getFilteredPaths"], undefined>;
+type CreateInstanceKeyPathsType = Parameters<GetFilteredPathsType>[0]["createInstanceKeyPaths"];
+type targetItemsType = Extract<Parameters<CreateInstanceKeyPathsType>[0], { targetItems: any }>["targetItems"];
```

Comment:
- Types should start with uppercase (e.g. `UseModelsTreeProps`).
- I think we could avoid the complexity of defining `TargetItemsType` and instead just define `targetItems: InstanceKey[]` in `CustomModelsTreeProps`. `InstanceKey`, AFAIK, is a public exported type.

Reply:
fixed